lib/link.js

```diff
@@ -11,8 +11,16 @@ export default class Link extends Component {
 
   static propTypes = {
     children: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.element
+      PropTypes.element,
+      (props, propName) => {
```

Comment:
That's interesting. I think they mean that throwing an Error is not going to stop the component from rendering, which it does if you don't use oneOfType. The mention to console.warn is weird though because the warning is shown on both client and server correctly 🤔